src/Build/Instance/ReflectableTaskPropertyInfo.cs

```diff
@@ -52,6 +52,17 @@ internal ReflectableTaskPropertyInfo(PropertyInfo propertyInfo)
             _propertyInfo = propertyInfo;
         }
 
+        internal ReflectableTaskPropertyInfo(PropertyInfo propertyInfo, bool output, bool required, bool IsAssignableToITaskItemType)
```

Comment:
Add comment that I'm overriding the default value we figure out for IsAssignableToITask